packages/ui-kit/src/Timeline/implementation/vis-timeline/useVisTimeline.ts

```diff
@@ -61,7 +60,7 @@ export function useVisTimeline(
       if (entry) {
         setTooltipEntry({
           entry: toTooltipEntry(entry),
-          position: { x: event.clientX, y: event.clientY },
+          target: event.target as Element,
```

Comment:
to avoid typecast you can:

```
    const onItemHover = ({ item, event }: { item: number; event: { target: Element } }) => {
      const entry = dataMapRef.current.get(item) || getClusterById(item, timeline);

      if (entry) {
        setTooltipEntry({
          entry: toTooltipEntry(entry),
          target: event.target,
        });
      }
    };
```